### DISCUSSION TITLE
Using goreleaser through curl fails at sha256sum checksum step

### DISCUSSION THREAD

**Discussion by AmanRiat1** [goreleaser/goreleaser#4190] [Q&A] [answered]
I'm using Goreleaser through the command `sh 'curl -sfL https://goreleaser.com/static/run | bash'` so that it can be used in a Jenkins pipeline. It fails at the sha256sum step with the following error message: 

```
sha256sum: unrecognized option '--ignore-missing'
```
For context this is on a CentOS 7 machine. I'm thinking it's just a problem with sha256sum just being outdated and a newer version should fix it. Just wanted to confirm or see if anyone else had this problem?

**Comment by caarlos0** [comment, marked as answer]
It probably is due to old sha256sum, yes
